### PR TITLE
Fix various code typos

### PR DESCRIPTION
**Pull request by Era-cell**

### CODE REVIEW

File: src/jsonschema/include/sourcemeta/jsontoolkit/jsonschema_transform_bundle.h [hunk 1]
```diff
@@ -39,7 +39,7 @@ namespace sourcemeta::jsontoolkit {
 ///                                const std::set<std::string> &vocabularies,
 ///                                const sourcemeta::jsontoolkit::Pointer
 ///                                  &pointer) const
-///       -> bool override
+///       -> bool override{
```
Comment (jviotti) [inline src/jsonschema/include/sourcemeta/jsontoolkit/jsonschema_transform_bundle.h:42]:
```suggestion
///       -> bool override {
```

File: src/jsonschema/walker.cc [hunk 2]
```diff
@@ -19,14 +19,14 @@ auto sourcemeta::jsontoolkit::keyword_priority(
 }
 
 namespace {
-enum class SchemaWalkerype_t { Deep, Flat };
+enum class SchemaWalkertype_t { Deep, Flat };
```
Comment (jviotti) [inline src/jsonschema/walker.cc:22]:
```suggestion
enum class SchemaWalkerType_t { Deep, Flat };
```

Let's do camelCase

**Comment by jviotti** [comment]
Oh great stuff. Thanks a lot for catching these. Some minor comments but in general looks great

**Comment by Era-cell** [comment]
> Oh great stuff. Thanks a lot for catching these. Some minor comments but in general looks great

@jviotti should I always create a new branch while working on different features else prev commits remain?

**Comment by jviotti** [comment]
Correct! You have to keep separate branches for each PR you are working on, as a PR is tied to a branch. Otherwise you will end up affecting unrelated PRs

**Review by jviotti — APPROVED** [review]

**Pull request merged**